client/components/inputTime.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+export default class InputTime extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: '13:30'
+    };
```

Comment:
You are going to need to get this value up to the form component. As the user fills out the form, the form should be accumulating all of their inputs into state.